<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DSS::AlignmentParameters</class>
 <widget class="QWidget" name="DSS::AlignmentParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alignment</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Transformation used during alignment</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QRadioButton" name="alignBisquared">
        <property name="text">
         <string>Bisquared</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="alignNone">
        <property name="text">
         <string>No alignment</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="alignBilinear">
        <property name="text">
         <string>Bilinear</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="5">
       <widget class="QLabel" name="text">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="alignBicubic">
        <property name="text">
         <string>Bicubic</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="alignAuto">
        <property name="text">
         <string>Automatic</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
